### PR TITLE
Model updates 2022-10-25

### PR DESCRIPTION
**Pull request by invisig0th**

### CODE REVIEW

File: synapse/models/media.py [hunk 1]
```diff
@@ -9,69 +9,79 @@ def getModelDefs(self):
...
                 ('publisher', ('ou:org', {}), {
-                    'doc': 'The organization which published the news.',
-                }),
+                    'doc': 'The organization which published the news.'}),
+
+
                 ('publisher:name', ('ou:name', {}), {
-                    'doc': 'The name the publishing org used to publish the news.',
-                }),
+                    'doc': 'The name the publishing org used to publish the news.'}),
```
Comment (therealsilence) [inline synapse/models/media.py:43]: Tiny typo - "The name OF the publishing..."

**Review by therealsilence — COMMENTED** [review]
Tiny typo

File: synapse/models/infotech.py [hunk 2]
```diff
@@ -1068,31 +1076,33 @@ def getModelDefs(self):
...
 
                     ('isos', ('bool', {}), {
                         'doc': 'Set to True if the software is an operating system.'}),
+
                     ('islib', ('bool', {}), {
                         'doc': 'Set to True if the software is a library.'}),
+
                     ('techniques', ('array', {'type': 'ou:technique', 'sorted': True, 'uniq': True}), {
-                        'doc': 'A list of techniques employed by the author of the software.'}),
+                        'doc': 'Deprecated for scalability. Please use -(uses)> ou:technique.'}),
```
Comment (Cisphyx) [inline synapse/models/infotech.py:1105]: Needs deprecated=True

File: synapse/models/orgs.py [hunk 3]
```diff
@@ -560,7 +566,7 @@ def getModelDefs(self):
                         'doc': 'An array of MX domains used by email addresses issued by the org.',
                     }),
                     ('techniques', ('array', {'type': 'ou:technique', 'sorted': True, 'uniq': True}), {
-                        'doc': 'A list of techniques employed by the organization.',
+                        'doc': 'Deprecated for scalability. Please use -(uses)> ou:technique.',
```
Comment (Cisphyx) [inline synapse/models/orgs.py:569]: Needs deprecated=True

File: synapse/models/risk.py [hunk 4]
```diff
@@ -81,8 +105,10 @@ def getModelDefs(self):
                     ('goals', ('array', {'type': 'ou:goal', 'sorted': True, 'uniq': True}), {
                         'doc': 'The assessed goals of the threat cluster activity.'}),
                     ('techniques', ('array', {'type': 'ou:technique', 'sorted': True, 'uniq': True}), {
-                        'doc': 'A list of techniques employed within the threat cluster.'}),
+                        'deprecated': True,
+                        'doc': 'Deprecated for scalability. Please use -(uses)> ou:technique.'}),
                 )),
+                ('risk:tool:taxonomy', {}, ()),
```
Comment (Cisphyx) [inline synapse/models/risk.py:111]: Needs a migration to populate for existing risk:tool:software:type values

**Review by mbvertex — DISMISSED** [review]

**Review by Cisphyx — APPROVED** [review]

**Pull request merged**